src/graphql/types/object/onchain-tx-fee.ts

```diff
@@ -1,12 +1,12 @@
 import { GT } from "@graphql/index"
-import ExpectedConfirmationTime from "../scalar/expected-confirmation-time"
+// import ExpectedConfirmationTime from "../scalar/expected-confirmation-time"
 import SatAmount from "../scalar/sat-amount"
 
 const OnChainTxFee = new GT.Object({
   name: "OnChainTxFee",
   fields: () => ({
     amount: { type: GT.NonNull(SatAmount) },
-    expectedConfirmationTime: { type: GT.NonNull(ExpectedConfirmationTime) },
+    // expectedConfirmationTime: { type: GT.NonNull(ExpectedConfirmationTime) },
```

Comment:
Shouldn't commit commented code. Since moving to the structure with `targetConfirmations` is this file even needed anymore?